util/envDynawo.sh

```diff
@@ -1629,6 +1614,9 @@ copy_sources() {
 }
 
 binary_rpath_for_darwin() {
+  if [ "`id -n -u`" = "dynawo" ]; then
+    error_exit "Your username cannot be dynawo to deploy a binary."
```

Comment:
Just curious: why not?

Reply:
I copy some files based on a regex later and if dynawo is the username it will mess up everything... So to avoid complicating things I added this protection. But it is only necessary on mac so I have to correct the position of the test